src/WebJobs.Extensions.DurableTask/DurableOrchestrationClientBase.cs

```diff
@@ -40,10 +40,22 @@ public abstract class DurableOrchestrationClientBase
         /// Creates a <see cref="HttpManagementPayload"/> object that contains status, terminate and send external event HTTP endpoints.
         /// </summary>
         /// <param name="instanceId">The ID of the orchestration instance to check.</param>
-        /// <returns>Instance of <see cref="HttpManagementPayload"/></returns>
+        /// <returns>Instance of the <see cref="HttpManagementPayload"/> class.</returns>
         public abstract HttpManagementPayload CreateHttpManagementPayload(string instanceId);
 
-        /// <inheritdoc />
+        /// <summary>
+        /// Creates an HTTP response which either contains a payload of management URLs for a non-completed instance
+        /// or contains the payload containing the output of the completed orchestration.
+        /// </summary>
+        /// <remarks>
+        /// If the orchestration instance completes within the specified timeout, then the HTTP response payload will
+        /// contain the output of the orchestration instance formatted as JSON. However, if the orchestration does not
+        /// complete within the specified timeout, then the HTTP response will be identical to that of the
```

Comment:
This overload of WaitForCompletionOrCreateCheckStatusResponseAsync doesn't take a timeout parameter, so should we change instances of "specified timeout" here to "30 seconds"?

Reply:
Good catch. I had a feeling that a blind copy/paste might get me into trouble...